app/features/ledger/ExportPrivateKeySeed.ts

```diff
@@ -20,14 +31,15 @@ function requestKeys(
     return transport.send(0xe0, INS_EXPORT_PRIVATE_KEY_SEED, p1, p2, data);
 }
 
-export async function getPrivateKeySeeds(
+export async function getPrivateKeys(
```

Comment:
The updated methods could use some documentation stating what the `version` variable does. Because if the method is called `getPrivateKeys`, then it is not obvious that if `version === 1` that it returns seeds and not the final keys.